File: app/src/main/java/io/github/jamilelima/pomofocus/adapters/TabsAssessorAdapter.java
package io.github.jamilelima.pomofocus.adapters;

import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import io.github.jamilelima.pomofocus.Fragments.TasksDoneFragment;
import io.github.jamilelima.pomofocus.Fragments.TasksPendingFragment;

public class TabsAssessorAdapter extends FragmentPagerAdapter {


    public TabsAssessorAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int i) {
        switch (i) {
            case 0:
                TasksPendingFragment tasksPendingFragment = new TasksPendingFragment();
                return tasksPendingFragment;
            case 1:
                TasksDoneFragment tasksDoneFragment = new TasksDoneFragment();
                return tasksDoneFragment;
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return 2;
    }

    @Nullable
    @Override
    public CharSequence getPageTitle(int position) {
        switch (position) {
            case 0:
                return "Pending";

            case 1:
                return "Done";

            default:
                return null;
        }
    }
}

File: app/src/main/java/io/github/jamilelima/pomofocus/adapters/TabsAssessorAdapter.kt
package io.github.jamilelima.pomofocus.adapters

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import io.github.jamilelima.pomofocus.fragments.TasksDoneFragment
import io.github.jamilelima.pomofocus.fragments.TasksPendingFragment

class TabsAssessorAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(i: Int): Fragment? {
        return when (i) {
            0 -> {
                TasksPendingFragment()
            }
            1 -> {
                TasksDoneFragment()
            }
            else -> null
        }
    }

    override fun getCount(): Int {
        return 2
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return when (position) {
            0 -> "Pending"

            1 -> "Done"

            else -> null
        }
    }
}
